cmd/nerdctl/exec.go

```diff
@@ -66,6 +66,10 @@ var execCommand = &cli.Command{
 			Aliases: []string{"e"},
 			Usage:   "Set environment variables",
 		},
+		&cli.StringSliceFlag{
+			Name:  "env-file",
+			Usage: "Set environment variables from file",
+		},
```

Comment:
Could you update the document?
https://github.com/containerd/nerdctl#whale-nerdctl-exec